framework/Steps.py

```diff
@@ -907,7 +907,7 @@ def _localInitializeStep(self,inDictionary):
                                                        okay = 'HDF5',
                                                        received = inDictionary['Output'][i].type))
       # from ROM model to ...
-      elif isinstance(inDictionary['Input'][i], Models.ROM):
+      elif isinstance(inDictionary['Input'][i], Models.ROM) or isinstance(inDictionary['Input'][i], Models.ExternalModel):
```

Comment:
Do you think it would be worth having a list of serializable models somewhere, maybe in the Models.Factory or Models.__init__?